docs/backend-api/resources/tracking/tracker/sensor/index.md

```diff
@@ -76,10 +89,22 @@ Contains API calls to interact with sensors.
 
 * `type` - string. Always "discrete".
 * `id` - int. An ID of a sensor.
-* `sensor_type` - [enum](../../../../getting-started.md#data-types). Type of the sensor.
+* `sensor_type` - [discrete sensor type](./index.md#Discrete-sensor-type-values). Type of the sensor.
 * `name` - string, max size 100.
 * `input_number` - int, [1..8]. Assigned input number.
 
+#### Discrete sensor type values
+
+*   `fuel`
+*   `temperature`
+*   `rpm`
+*   `custom`
+*   `fuel_consumption`
+*   `instant_consumption`
+*   `power`
+*   `speed`
+*   `flow_meter`
+*   `acceleration`
```

Comment:
This is an incorrect list. Please look at what elements are in the UI.

![image](https://github.com/SquareGPS/navixy-api/assets/20779569/f3602bca-4add-4481-858f-d2760fb2f861)